torch/distributed/checkpoint/storage_utils.py

```diff
@@ -20,6 +20,7 @@ def _storage_setup(
             "storage_reader/storage_writer is None."
         )
 
+    targets: List[type[Union[StorageReader, StorageWriter]]] = []
```

Comment:
might want to stick with typing.Type for now for checkers on older version of Python